gc/base/MarkingScheme.hpp

```diff
@@ -139,13 +141,7 @@ class MM_MarkingScheme : public MM_BaseVirtual
 	MMINLINE bool
 	inlineMarkObject(MM_EnvironmentBase *env, omrobjectptr_t objectPtr, bool leafType=false)
 	{
-		bool didMark = false;
```

Comment:
I am not sure about these changes.  They make the code harder to read. Why did you want to change this code?